integration/constants.go

```diff
@@ -3,16 +3,15 @@ package integration
 // Tracks the start ports handed out to different tests, in a bid to minimise conflicts.
 const (
 	StartPortGethNetworkTest         uint64 = 30000
-	StartPortWalletExtensionTest     uint64 = 31000
```

Comment:
This was unused. It should have been removed when I deleted the wallet extension integration tests.